tsconfig.json

```diff
@@ -33,8 +34,9 @@
     ]
   },
   "include": [
-    "**/*.ts"
-  ],
+    "**/*.ts",
+    "src/**/*"
```

Comment:
why? are there any non `*.ts` files that are actually typescript?

Reply:
No, but there are still .js files and they will not be copied to /dist/ unless statically imported (i.e. const a = require('runRelay.js') will not make runRelay.js be copied to 'dist'